shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsController.mm

```diff
@@ -577,6 +565,9 @@ - (void)clipViewSetMaskView:(UIView*)clipView {
 - (void)applyMutators:(const flutter::MutatorsStack&)mutatorsStack
          embeddedView:(UIView*)embeddedView
          boundingRect:(const SkRect&)boundingRect {
+  // Becomes NO if Apple's API changes and blurred backdrop filters cannot be applied.
+  static BOOL sCanApplyBlurBackdrop = YES;
```

Comment:
is the `s` prefix a common practice

Reply:
Will double-check but my recollection was that the style guide recommends a k prefix for constants, a g prefix for mutable globals, and an s prefix for statics. I believe we enforce naming on constants and mutable globals in our .clang-tidy rules. Will double check when I’m back at a computer.

Reply:
I thought this too, but it looks like it's not in the style guide.

Reply:
I suppose the right thing to do here is to just make this an ivar. The whole point appears to be just to avoid the overhead of trying to create backdrop filters if we've ever failed to create one in the past.

This class will only ever have the one instance created and held by the engine for the duration of its life. Most apps only ever have one engine and the add-to-app cases with multiple engines either at once or over the course of an app's lifetime are already taking a far bigger hit than those being saved by this bool.

Mostly, this approach saves me scratching my head trying to remember whether the `s` prefix rule was real or imagined.